packages/vite-plugin-cloudflare/src/cloudflare-environment.ts

```diff
@@ -155,17 +155,12 @@ export function createCloudflareEnvironmentOptions(
 					input: userConfig.root
 						? path.resolve(userConfig.root, options.main)
 						: path.resolve(options.main),
-					external: [...cloudflareBuiltInModules, ...getNodeCompatExternals()],
+					external: [...getNodeCompatExternals()],
```

Comment:
@petebacondarwin we set these node externals unconditionally even if the worker is not under nodejs_compat, is that ok?

Or should we do it conditionally? (similarly to the [`addNodeBuiltinsIfNeeded` function](https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/52/files#diff-0217c39a7be6fcdb410ac8dfea2ae2998b882a1aaba3f2e83ff2831a0beee7bfR222) that I'm adding here)

Reply:
We can do it conditionally if you like. I was being a bit lazy perhaps?